dht.go

```diff
@@ -280,7 +280,8 @@ func makeDHT(ctx context.Context, h host.Host, cfg config) (*IpfsDHT, error) {
 	}
 
 	// construct routing table
-	rt, err := makeRoutingTable(dht, cfg, maxLastSuccessfulOutboundThreshold)
+	// use twice the theoritical usefulness threhold to keep older peers around longer
+	rt, err := makeRoutingTable(dht, cfg, 2*maxLastSuccessfulOutboundThreshold)
```

Comment:
@aarshkshah1992 what do you think about having the usefulness here be an option so that we can play around with it in testground?

Perhaps something like an option that takes `func(refreshThreshold time.Duration) time.Duration`?

Reply:
@aschmahmann #654